es/lecciones/corpus-paralelo-lfaligner.md

```diff
@@ -0,0 +1,278 @@
+
+Un corpus paralelo o *bitexto* consiste en la recopilación de varias versiones de un texto. En este tutorial aprenderás a alinear el texto original con sus traducciones para poder cotejarlos con facilidad.
+
+## Introducción
+LF Aligner es un programa gratuito, basado en un [algoritmo de código abierto de alineación de oraciones](https://github.com/danielvarga/hunalign), que pertenece al conjunto de herramientas digitales llamadas ***CATs*** (*Computer Assisted Translation Tools*, por sus siglas en inglés) o herramientas de traducción asistida. Principalmente, se usa para la creación de bitextos que facilitan la búsqueda de términos especializados y sus traducciones. Sitios como [Linguee](https://www.linguee.es/) utilizan este tipo de herramientas para crear enormes corpus paralelos que el usuario puede consultar fácilmente. En ciencias sociales y humanidades podemos aprovechar este programa para crear textos que faciliten las tareas de lectura distante y [análisis estilístico](http://vocabularios.caicyt.gov.ar/portal/index.php?task=fetchTerm&arg=134&v=42). La aplicación puede importar texto de documentos en múltiples formatos y de memorias de traducción generadas con programas de código libre o privativo. En este tutorial nos centraremos en la importación de texto de fuentes digitales usadas comunmente por los investigadores como páginas web o documentos de texto plano, ya que, además, agilizan el proceso de alineación del corpus.
+
+Para este tutorial necesitarás los siguientes materiales y conocimientos:
+---
+
+* El programa [LF Aligner](https://sourceforge.net/projects/aligner/), disponible para Windows (versión 4.2), Mac (versión 3.12) y Linux (versión 3.11). **En este tutorial nos centraremos en la versión de Windows, que es la más reciente. Sin embargo, también se explicará cómo utilizarlo en Mac y en sistemas basados en el kernel de Linux.** La interfaz del programa está en inglés y no cuenta con una versión en español, por lo que se proveen traducciones de algunos elementos que son indispensables para comprender el funcionamiento de LF Aligner.
+* Un texto de partida -digitalizado- y por lo menos una traducción de este. En este caso, alinearemos distintas traducciones de un documento que desde 1948 guía el quehacer y la convivencia humana en todos los ámbitos de la vida pública y privada, la [Declaración Universal de Derechos Humanos](https://es.wikipedia.org/wiki/Declaraci%C3%B3n_Universal_de_los_Derechos_Humanos): en [español](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-paralelo-lfaligner/DDHH_es.txt), [inglés](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-paralelo-lfaligner/DDHH_en.txt), [francés](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-paralelo-lfaligner/DDHH_fr.txt) y [portugués](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-paralelo-lfaligner/DDHH_pt.txt)
```

Comment:
Please turn these into relative links pointing to `/assets/corpus-paralelo-lfaligner...`